application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -25,7 +25,7 @@
      * {@link Guild#ban(User, int, String)}.
      */
     private static final int REASON_MAX_LENGTH = 512;
-    private static final Color AMBIENT_COLOR = Color.decode("#895FE8");
+    static final Color AMBIENT_COLOR = Color.decode("#895FE8");
```

Comment:
What is the point of having it inModerationUtils. What if you want to use that colour for a non moderation command. If you want to keep it their maybe making it public would be better.

Reply:
I do not understand what your suggestion, sorry.

The color was already in `ModerationUtils` beforehand. I just changed it from `private` to package-visible. So that I can use it in `/audit` to streamline the color used by moderation commands...

Reply:
oh ok